### PR TITLE
Feature Request for kill-process back to root search after process killed

### PR DESCRIPTION
**Pull request by RakerZh**
## Description

I add preference option for the kill-process extension that user can choose to back to root search after process killed.
## Screencast
Screenshot 2024-02-26 at 15 19 10

https://github.com/raycast/extensions/assets/76530956/158a7959-905f-4431-ae64-df96d8ced983

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by zhenpewu** [comment]
why not configure to the pop to root preference in the Advance settings directly?

**Comment by pernielsentikaer** [comment]
> why not configure to the pop to root preference in the Advance settings directly?

I can see why it could become handy to set here, @rolandleth what do you think?

**Comment by pernielsentikaer** [comment]
@RakerZh, could you also add a changelog entry and make the CI happy 🙂

Let me know if you need help 🙂

File: extensions/kill-process/package.json [hunk 1]
```diff
@@ -101,6 +101,14 @@
       "default": true,
       "type": "checkbox"
     },
+    {
+      "description": "Back to root search after killing a process",
+      "label": "Back to Root Search",
```
Comment (rolandleth) [inline extensions/kill-process/package.json:106]: I would rename these two as "Go back to..."

File: extensions/kill-process/package.json [hunk 2]
```diff
@@ -101,6 +101,14 @@
...
+      "name": "backRootAfterKill",
```
Comment (rolandleth) [inline extensions/kill-process/package.json:107]: This too, maybe `goToRootAfterKill`.

File: extensions/kill-process/src/index.tsx [hunk 3]
```diff
@@ -28,6 +29,7 @@ export default function ProcessList() {
   const refreshDuration = +preferences.refreshDuration;
   const closeWindowAfterKill = preferences.closeWindowAfterKill;
   const clearSearchBarAfterKill = preferences.clearSearchBarAfterKill;
+  const backRootAfterKill = preferences.backRootAfterKill;
```
Comment (rolandleth) [inline extensions/kill-process/src/index.tsx:32]: This as well.

File: extensions/kill-process/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Kill Process Changelog
 
+## [Back Root Search After Process Kiled] - 2024-03-01
+- Added preference to back to root search after process killed
```
Comment (rolandleth) [inline extensions/kill-process/CHANGELOG.md:4]: "... to go back to..."
"... after a process is killed"

**Review by rolandleth — CHANGES_REQUESTED** [review]
Just a few wording improvements

File: extensions/kill-process/package.json (hunk 1 above)
Comment (zhenpewu) [reply extensions/kill-process/package.json]: does "Pop to Root Search" make more sense to make consistent with the wording in the Advanced Raycast Setting.

**Comment by pernielsentikaer** [comment]
Is this ready for another review @RakerZh

**Comment by RakerZh** [comment]
> Is this ready for another review @RakerZh

I think so

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**